packages/OreIdWalletConnect/src/types.tsx

```diff
@@ -27,14 +27,17 @@ export enum ChainNetwork {
   DspEosKylin2 = 'kylin-dsp-2.liquidapps.io',
   DspMoonlighting = 'eos_moon_blockstartdsp_com',
   DspMoonlightingTest = 'eos_moontest_blockstartdsp_com',
+  // DOS
+  DosMain = 'dos_main',
+  DosTest = 'dos_test',
   // ETH
-  EthMain = 'eth_main', // 1
-  EthRopsten = 'eth_ropsten', // 3
-  EthRinkeby = 'eth_rinkeby', // 4
-  EthGoerli = 'eth_goerli', // 5
+  EthMain = 'eth_main',
```

Comment:
We shouldn't be dropping the comments with the chain Ids - if anything we should be adding them to the standard shared model type.

Put the comments back and use these chainIds

  // DOS
  DosMain = 'dos_main', // 7979
  DosTest = 'dos_test', // 1311